Signal/src/view controllers/UITests/SignalsViewController.m

```diff
@@ -88,7 +88,7 @@ -(void)viewWillAppear:(BOOL)animated
 - (void)viewDidAppear:(BOOL)animated{
     [super viewDidAppear:animated];
     
-    if (![TSAccountManager isRegistered]){
+    if (![TSAccountManager isRegistered] && ![Environment.preferences getIsMigratingToVersion2Dot0]){
```

Comment:
this was one place that the migration was broke; when the tab view controller was removed this became the first view and then would display the signup flow as migration was in progress!